MKS - Backing up with Velero - Added information of install newer versions from 1.14 (#7182)
ovh/docs: pages/public_cloud/containers_orchestration/managed_kubernetes/backing-up-cluster-with-velero/guide.en-ie.md

  --snapshot-location-config region=<public cloud region without digit>,enableSharedConfig=true
```

> [!primary]
>

> > Starting with version 1.14 the plugin-for-csi is integrated in velero. We can simply remove it from the install example if we install a version 1.14 or newer. For upgrading an older version follow the upgrade notes [Upgrade-to-1.14](https://velero.io/docs/v1.14/upgrade-to-1.14/).

Reviewer: ```suggestion
> Starting with version 1.14 the plugin-for-csi is integrated in velero. You can simply remove it from the install example if you install version 1.14 or newer. For upgrading an older version follow the upgrade notes: [Upgrade-to-1.14](https://velero.io/docs/v1.14/upgrade-to-1.14/).
```
